controllers/space/mapper.go

```diff
@@ -0,0 +1,28 @@
+package space
+
+import (
+	"errors"
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+var mapperLog = ctrl.Log.WithName("NSTemplateSetToSpaceMapper")
+
+func MapNSTemplateSetToSpace(hostNamespace string) func(object client.Object) []reconcile.Request {
+	return func(obj client.Object) []reconcile.Request {
+		if nsTemplateSet, ok := obj.(*toolchainv1alpha1.NSTemplateSet); ok {
```

Comment:
We don't have to cast the object and check if it's really `NSTemplateSet` - the watcher ensures that the mapper is triggered only for NSTemplateSets, but that's probably just a detail

Reply:
fair enough :+1: